modules/perception/obstacle/camera/tracker/BUILD

```diff
@@ -3,18 +3,16 @@ load("//tools:cpplint.bzl", "cpplint")
 package(default_visibility = ["//visibility:public"])
 
 cc_library(
-    name = "perception_obstacle_camera_tracker",
-    srcs = [
-        "mix_camera_tracker_util.cc"
-    ],
-    hdrs =  glob(["*.h"]),
-    deps = [
-        "//modules/perception/lib/base",
-        "//modules/perception/obstacle/common:perception_obstacle_common",
-        "//modules/perception/obstacle/base:perception_obstacle_base",
-        "//modules/perception/obstacle/camera/common:perception_obstacle_camera_common",
-        "@eigen"
-    ],
-)
+  name = "perception_obstacle_camera_tracker_cascaded",
```

Comment:
recommend to rename perception_obstacle_camera_tracker_cascaded with camera_tracker_cascaded. perception_obstacle info is available in directory path.

Reply:
Will do. Thanks!